lib/features/recurring_donations/overview/widgets/create_recurring_donation_button.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+
+class CreateRecurringDonationButton extends StatelessWidget {
+  const CreateRecurringDonationButton({required this.onClick, super.key});
+
+  final void Function() onClick;
```

Comment:
You cna replace these functions with VoidCallback instead of "void Function()", i think it's better